tests/utils.go

```diff
@@ -1510,18 +1510,23 @@ func CreateVmiOnNode(vmi *v1.VirtualMachineInstance, nodeName string) *v1.Virtua
 	return CreateVmiOnNodeLabeled(vmi, util2.KubernetesIoHostName, nodeName)
 }
 
-// RunCommandOnVmiPod runs specified command on the virt-launcher pod
-func RunCommandOnVmiPod(vmi *v1.VirtualMachineInstance, command []string) string {
-	virtClient := kubevirt.Client()
-
+func GetVmiPod(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) *k8sv1.Pod {
```

Comment:
We already have [GetPodByVirtualMachineInstance()](https://github.com/kubevirt/kubevirt/blob/v1.0.0-beta.0/tests/libvmi/status.go#L17) for that.

Reply:
But would be great to use it inside RunCommandOnVmiPod()!

Reply:
Thanks! I will use `GetPodByVirtualMachineInstance` but I prefer to refactor `RunCommandOnVmiPod` in a separate PR